backend/hmnet/syncing/discovery_test.go

```diff
@@ -0,0 +1,41 @@
+package syncing
+
+import (
+	"seed/backend/hmnet/syncing/rbsr"
+	"seed/backend/storage"
+	"seed/backend/util/colx"
+	"seed/backend/util/sqlite"
+	"seed/backend/util/sqlite/sqlitex"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadExternalStore(t *testing.T) {
+	t.Skip("This uses external database for local test only")
+	t.Parallel()
+	pool := loadLocalDB(t, "/home/julio/.config/Seed-local/daemon/db/db.sqlite")
```

Comment:
@juligasa I guess this test doesn't need to be here now?

BTW, do we have any proper tests for the new push functionality?